tests/test_clitool.py

```diff
@@ -112,6 +112,68 @@ def test_parse_single_project_json_as_expected(self):
         assert projects[0]['metadata']['title'] == node['attributes']['title']
         assert isinstance(projects[0]['files'], list)
 
+# Commenting out this test as it needs fixing
```

Comment:
Do we have an issue for this bug? If so, can you mention the issue number in the comment please, then it's clear to anyone reading the code how to find out if it is fixed.